recipes/date/all/conanfile.py

```diff
@@ -23,7 +31,46 @@ def source(self):
         for patch in self.conan_data["patches"][self.version]:
             tools.patch(**patch)
 
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def _configure_cmake(self):
+        cmake = CMake(self)
+        cmake.definitions["ENABLE_DATE_TESTING"] = False
+        if self.settings.os == "Windows":
+            cmake.definitions["USE_TZ_DB_IN_DOT"] = False
+            cmake.definitions["USE_SYSTEM_TZ_DB"] = False
+        else:
+            cmake.definitions["USE_TZ_DB_IN_DOT"] = self.options.use_tz_db_in_dot
+            cmake.definitions["USE_SYSTEM_TZ_DB"] = self.options.use_system_tz_db
+        cmake.configure(source_folder=self._source_subfolder)
```

Comment:
```suggestion
        cmake.configure()
```

This will fix the cmake configuration, including Visual Studio build.